fixtures/README.md

```diff
@@ -0,0 +1,40 @@
+# Fixtures Files
+
+## Purpose
+
+We have collected various JS files from a variety of public open source projects on GitHub.
+
+These files help us ensure that future updates to this ESLint config/plugin do not break existing code that passes linting. When a new rule is created, or when an existing rule is updated, these files will help make us aware of scenarios where the updates do not work as expected.
+
+## What Happens If a Plugin/Config Update Breaks Fixtures?
+
+- If the rule is catching something that is not actually a problem:
+  - Reconfigure, disable, or modify the lint rule
```

Comment:
Is there any process for amending or refining rules, e.g.  based on this process? As an example, I used this to import files from Koa as fixtures, and in one instance, a class getter was flagged for returning (`undefined`) in some cases. This flag would allow that getter to pass muster: `/* eslint getter-return: ["error", { allowImplicit: true }] */`. How would we debate this modification, for example?

Reply:
Are you asking how we can update our lint rules? I think we can just follow the normal process here, by opening issues/pr's on this repo and on our `guides` repo

Reply:
OK, so at the time of adding the fixture, if I choose to solve the issue by adding a comment, is that an indication that we need to review the rule? Should we have any documentation to that effect? Just trying to understand how we're using this process to inform updates to the ruleset. And I'm asking to have a little more in the documentation here about suggested follow-up, if that's ok.

Reply:
I think that is a question that would need to be answered case-by-case.

When I was importing babel files, there were a few cases where the `complexity` rule was triggered. When that rule is triggered, that is supposed to mean that the function that triggers it is probably a good option for breaking it into smaller functions. In babel's case, that rule was triggering correctly, and it didn't seem unreasonable to me, so I disabled the rule in that file via a comment.

For `getter-return` which you saw, I do not see a reason to change the rule configuration. I don't like getters generally, so I am suspicious of any code that uses getters 😄. I think it is reasonable to expect all getters to return a value.

Reply:
OK, I see I'm overthinking the process, a bit. Really doesn't matter how you deal with the rule-breaking in a fixture, since in the end, it means any file we keep will have to pass (either because rule-breaking code is bypassed or removed).

For ES classes, I think of getters and setters as just basic elements of OOP, and the fact that this particular getter could return either `undefined` or a value was a choice I wondered about. Like you say, I guess if the person adding the fixture thinks an issue that's caught is worth a team review, we can create a GH issue (or PR, but maybe an issue is better) to go through that process.

I guess this isn't really the point of adding fixtures, but when trying out this update, it made me think that it's a potential way to consider what rules other (hopefully respected) codebases abide by.

No need to discuss further here, and thanks for the feedback.